src/app/not-found.tsx

```diff
@@ -0,0 +1,14 @@
+import { Text } from "@/components/ui/text"
+
+const Custom404 = ({ errorMessage }: { errorMessage: string }) => {
+  return (
+    <div className="flex h-screen">
+      <div className = "m-auto flex flex-col items-center">
```

Comment:
pls add more space between the text here

Reply:
it is a nit, but I do think the title is crunched too close to the error message text